swagger.yaml

```diff
@@ -621,6 +621,14 @@ paths:
           in: query
           type: boolean
           description: Return only reservations that are related to resource that either is or is not favorited by user, based on the boolean value.
+        - name: user
+          in: query
+          type: string
+          description: Return only reservations that are related to given user
```

Comment:
This could specify better what the filter expects. It needs the UUID of the user to work.